portfolio.css

```diff
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: 'Inter';
+  src: url(./assets/fonts);
+}
+
+.header-welcome {
+  padding-top: 39px;
+  color: #28352f;
+  padding-left: 24px;
+}
+
+.menu-icon {
+  position: absolute;
+  width: 18px;
+  height: 20px;
+  bottom: 6641px;
+  right: 35px;
+  top: 62px;
+  left: 322px;
+}
+
+.introduction-text {
+  font-size: 40px;
+  text-align: center;
+}
+
+.dark-blue {
+  color: #172b4d;
+}
+
+.main-page {
+  height: 100vh;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url(./assets/header-files/header-illsutration-mobile.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+}
```

Comment:
Excellent work given a height of 100vh 🥇 
Excelent work giving a display flex to this container 🥇 

- [ ] Kindly change the size of your background to cover, so it will take the entry size of your screen, as you now have only this able to 340px. With the `cover` property you will be responsive on more than 340px width.